package.json

```diff
@@ -202,4 +202,4 @@
     "webpack-manifest-plugin": "2.2.0",
     "webpack-merge": "4.2.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
Why is this change in here?

Reply:
I had to reinstall `typedock` and then manually removed it from the `package.json`

Reply:
I bet my IDE applied to autoformatting after that

Reply:
Next time please remove the change before merging.